view/templates/field_yesno.tpl

```diff
@@ -1,16 +0,0 @@
-
-	<div class="field yesno">
-		<label for="id_{{$field.0}}">{{$field.1}}</label>
-		<div class="onoff" id="id_{{$field.0}}_onoff">
```

Comment:
How about replacing this "div" with a "real" checkbox HTML tag and customize the look&feel like a toggle?
So we would have
- The toggle stay alive
- A screenreader can read it as a checkbox for correct parsing

Reply:
Like this one:
https://www.bootstraptoggle.com/
It is used as a checkox instead of our "div" construct, so there won't be that problems
@rabuzarus what do you say?